tiled/client/base.py

```diff
@@ -370,6 +380,99 @@ def update_metadata(self, metadata=None, specs=None):
 
         This is subject to change or removal without notice
 
+        Parameters
```

Comment:
Would it be useful to expand the main description in the docstring a little bit. I guess, it's fairly obvious what the function is doing just from it's name, so maybe just emphasising that the logic here is the same as in the usual `old_dict.update(new_dict)`, i.e. some values would be overwritten or added, but none of them would be removed, whereas `replace_metadata` can lead to some key-value pairs being dropped. (At least this is my understanding of how it works; am I right?)

The distinction between `update_` and `patch_` metadata is less obvious, I guess.

Reply:
I agree. I'll slightly expand the main description and add more details where appropriate.